### ISSUE TITLE
cannot run gqlgen init

### ISSUE THREAD

**Issue by bugzpodder** [99designs/gqlgen#432]
> gqlgen init
exec plan failed: required package was not loaded: github.com/99designs/gqlgen/graphql/introspection.Field

I followed instructions in:
https://github.com/99designs/gqlgen/pull/416

go run scripts/gqlgen.go init
exec plan failed: required package was not loaded: github.com/99designs/gqlgen/graphql/introspection.Schema
exit status 1

What am I missing?

**Comment by vektah** [comment]
I think you've hit some version shear, because those docs havent been merged into a tagged release yet you probably need to add something like this to your `Gopkg.toml`
```
[[constraint]]
  name = "github.com/99designs/gqlgen"
  branch = "master"
```

**Comment by bugzpodder** [comment]
I have the same issue in the current release by using the gqlgen binary.  
```
>gqlgen init
exec plan failed: required package was not loaded: github.com/99designs/gqlgen/graphql/introspection.Field
```

I also tried go build -o gqlgen vendor/github.com/99designs/gqlgen/main.go and it shows a similar error

**Comment by bugzpodder** [comment]
I don't have dep installed on my system, and I dont have a Gopkg.toml file.  I am using govendor to pull in https://github.com/kardianos/govendor and running go install to generate gqlgen binary.

**Comment by mathewbyrne** [comment]
Yeah this is a chicken and egg problem you get with codegen.  Basically the binary doesn't need runtime packages to generate your gqlgen server, but as part of the `init` the generated code is read in order to generate resolvers.  The way to get around it is probably to try and run `dep ensure` against the `generated.go` file, then try and run `gqlgen gen`.

**Comment by bugzpodder** [comment]
Just leaving a note, I was able to get govendor to work with gqlgen in the end (instead of dep).  Just a note that `gqlgen init` doesn't quiet work, you'll need to use dep to bootstrap the project, but afterwards you can just run gqlgen to regenerate the code without relying on dep.
